pages/content/amp-dev/documentation/guides-and-tutorials/develop/custom-javascript-tutorial.md

```diff
@@ -0,0 +1,329 @@
+      return passed ? checkMet(item.text) : checkRemoved(item.text);
+    });
+ }
+    // is called when user types in input
+    el.addEventListener("keyup", checkPassword);
+    // is called if user pastes into input 
+    el.addEventListener("change", checkPassword);
+};
+```
+
+
+
+Both of our check functions add and remove classes from the password requirement list items within the `rules` div. The `checkFailed` class will make our text red and the `checkPassed` will change it to green. 
+
+Just like non-AMP pages, once the `amp-script` logic adds or removes a class it will be reflected in the DOM. These classes are treated the same as all other CSS that don't work with `amp-script`. 
+
+For this to work, we will need to add the `checkFail` class to all the items and define both `checkPass` and `checkFailed` within the `<style amp-custom>` tag in the page head. 
+
+
+```html
+<ul>
+    <li id="lowercase" class="checkFail">Lowercase letter</li>
+    <li id="capital" class="checkFail">Capital letter</li>
+    <li id="number" class="checkFail">Number</li>
+    <li id="special" class="checkFail">Special character (@$!%*?&)</li>
+    <li id="eight" class="checkFail">At least 8 characters long</li>
+</ul> 
+```
+
+
+
+```css
+.checkPass {
+    color: #2d7b1f;
+} 
+.checkFail {
+    color:#c11136;
+}
+```
+
+
+Refresh the page and type into the password input. The elements corresponding the check passed or failed should turn red or green accordingly. However, we don’t signal when all checks have been passed and don’t account for the length... yet.
+
+<figure class="alignment-wrapper margin-">
+  <amp-video layout="intrinsic" poster="/static/img/docs/guides/storiesbp/do-background-still.jpg" width="400" height="720" loop autoplay noaudio>
+    <source src="/static/img/docs/tutorials/custom-javascript-tutorial/checkpassfail.mp4" type="video/mp4" />
+  </amp-video>
+</figure>
+
+
+## Communicate success 
+
+Now that we can signal to users what checks have been passed or failed, we can create a visual cue to let them know their password is complete. Add the following if statement to the `checkPassword` function:
+
+
+```js
+ const checkPassword = () => {
+    let failed = false;
+    let successTest = passwordChecks.items.filter((item) => {
+      let passed = item.checkRegEx(el.value);
+      // calls checkMet or checkRemoved
+      return passed ? checkMet(item.text) : checkRemoved(item.text);
+    });
+    // if all four checks are met and password is 8 or more char
+    // successTest.length will equal 4
+    if (successTest.length == 4 && el.value.length >= 8) {
+      // changes length check text color
+      checkMet(lengthCheck);
+      // enables submit button 
+      submitButton.setAttribute("class", "");
+    }
+  };
+```
+
+
+Then add the following `unclick` class to the `<style amp-custom>` tag and assign it to the form submit button in `index.html`. 
+
+
+```html
+<style amp-custom>
+  . . .
+   unclick {
```

Comment:
s/unclick/.unlick

but as mentioned above it's better to use the `disabled` attribute.

Reply:
Decided to just remove it, since it will likely confuse people...